src/groupTypes.ts

```diff
@@ -33,20 +33,13 @@ export interface Scalar {
     serialize(): Uint8Array
 }
 
-export interface ScalarCons {
-    size(g: Group): number
-    deserialize(g: Group, bytes: Uint8Array): Scalar
-}
-
 export interface Elt {
     g: Group
 
     isIdentity(): boolean
 
     isEqual(a: Elt): boolean
 
-    neg(): Elt
```

Comment:
@armfazh 

I noticed Element negate isn't actually used here ?

Reply:
The reason I brought it up is that it doesn't seem the noble `RistPoint`/`DecfaPoint` classes impl negate(). We could patch noble, but if they're not actually needed ?

Reply:
if not used, it can be removed. Also, it's not difficult to implement negation.

Reply:
Yes, it shouldn't be hard. 
May as well just do it, eh?